subsys/net/lib/nrf_cloud/src/nrf_cloud_agps.c

```diff
@@ -75,51 +74,51 @@ static int json_add_types_array(cJSON *const obj, enum gps_agps_type *types,
 }
 #endif /* IS_ENABLED(CONFIG_NRF_CLOUD_MQTT) */
 
-int nrf_cloud_agps_request(const struct gps_agps_request request)
+int nrf_cloud_agps_request(const struct nrf_modem_gnss_agps_data_frame request)
```

Comment:
not new to this PR, but is there a reason that the request parameter is not a pointer?

Reply:
I'm guessing it was an overlook in the original PR adding it, and then it's just lived on unchanged

Reply:
I'm guessing it was an overlook in the original PR adding it, and then it's just lived on unchanged

Reply:
Sure, if you don't mind. I don't see a reason to not have it as a pointer

Reply:
Done.